src/hotspot/share/opto/type.cpp

```diff
@@ -4036,15 +4037,23 @@ int TypeInstPtr::hash(void) const {
 //------------------------------dump2------------------------------------------
 // Dump oop Type
 #ifndef PRODUCT
-void TypeInstPtr::dump2( Dict &d, uint depth, outputStream *st ) const {
+void TypeInstPtr::dump2( Dict &d, uint depth, outputStream* st ) const {
```

Comment:
While you are at it, please also remove the excess whitespace after `(` and before `)`.

Reply:
done.